.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!-- Please search for existing issues to avoid creating duplicates. -->
+
+<!-- Use Help > Report Issue to prefill these. -->
+- Firebase Swissknife Version:
+- Browser:
+- Browser Version:
```

Comment:
Browser & Browser Version is irrelevant, you can ask for`node` version & `firebase-functions` version